Meta/shell_include.sh

```diff
@@ -12,7 +12,7 @@ die() {
 }
 
 exit_if_running_as_root() {
-    if [ "$(id -u)" -eq 0 ]; then
```

Comment:
Any docker-based workflow should absolutely be creating a non-root user.

Reply:
Addressed in the other comment.